### PR TITLE
Added some usefull extensions

### PR DESCRIPTION
**Pull request by scoreyou**

### CODE REVIEW

File: LeadKit/Sources/Extensions/Array/Array+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+//  THE SOFTWARE.
+//
+
+public extension Array where Element: Equatable {
+
+    /// Union array with another arrays, without element duplication
+    func union(values: [Array.Element]...) -> Array {
+        var result = self
+
+        for array in values {
```
Comment (igorkislyuk) [inline LeadKit/Sources/Extensions/Array/Array+Extensions.swift]: Can it be replaced with `forEach`?

Comment (petropavel13) [inline LeadKit/Sources/Extensions/Array/Array+Extensions.swift]: what for?

Comment (igorkislyuk) [inline LeadKit/Sources/Extensions/Array/Array+Extensions.swift]: We try remove `for ... in ...` in our code, isn't it?

Comment (petropavel13) [inline LeadKit/Sources/Extensions/Array/Array+Extensions.swift]: Only when it dramatically reduce amount of code.
for example:

```swift

let items = ["1", "2", "3"]
let labels = [lbl1, lbl2, lbl3]

for (item, label) in zip(items, label) {
    label.text = item
}
```


```swift
zip([lbl1, lbl2, lbl3], ["1", "2", "3"]).forEach { $0.0.text = $0.1 }
```

But `for` loop is more performance efficient. For unknown array size it's better to use `for` loop

Comment (igorkislyuk) [inline LeadKit/Sources/Extensions/Array/Array+Extensions.swift]: why `for` is more effective?

**Pull request merged**